wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -47,14 +47,72 @@ When adding redirects for a new or existing article, keep in mind that they shou
 
 The osu! wiki repository uses [continuous integration](https://docs.github.com/en/actions/guides/about-continuous-integration) (CI) to automatically check incoming pull requests for various common errors. The list of checks is configured in the [`continuous-integration.yml`](https://github.com/ppy/osu-wiki/blob/master/.github/workflows/continuous-integration.yml) file. 
 
-The [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file lists all plugins used by the CI, from which some were written by the osu! wiki maintainers.
+The [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file lists all (remark) plugins used by the CI, of which some were written by the osu! wiki maintainers.
 
 The CI checks are run automatically on every commit of a recurring contributor. In order to have their pull requests merged, contributors are expected to fix errors reported by the CI. To see the [status of checks](img/ci-status.png), do the following:
 
 1. Scroll down the pull request's page, find the `osu-wiki continuous integration` status row, and click the `Details` link.
 2. On the new page, expand the `run remark on changed files` step. Each finding is accompanied by its exact location in a file and a short description of why it is an error.
 
-If you need help with decrypting CI check errors, or fixing issues, ask in the `#osu-wiki` channel on Discord.
+Some errors also appear as annotations on the `Files changed` tab below the line where the issue was found.
+
+If you need help with decrypting CI check error messages, or fixing issues, ask in the `#osu-wiki` channel on Discord.
+
+#### Bypassing CI checks
+
+CI checks ensure that changes to the wiki keep it in a consistent state, but ways to bypass them exist as a fail safe, such that pull requests can still be merged in the event of false errors or discovered bugs in the checks. There are a few situations where intentionally bypassing a CI check is acceptable, as outlined below. Contact a [maintainer](/wiki/osu!_wiki/Maintenance/List_of_maintainers) if you need to bypass a check for a reason not mentioned here.
```

Comment:
I feel the first sentence is trying to say too much (=overloaded), mind simplifying or at least chopping it down a bit?

Reply:
perhaps like this?

```suggestion
CI checks normally prevent pull requests with errors from getting merged. However, in the event of false errors or discovered bugs in the checks, they can still be bypassed as a fail safe. There are a few situations where intentionally bypassing a CI check is acceptable, as outlined below. Contact a [maintainer](/wiki/osu!_wiki/Maintenance/List_of_maintainers) if you need to bypass a check for a reason not mentioned here.
```

Reply:
yes!